app/views/mailers/notify_mailer/unread_notifications_email.html.erb

```diff
@@ -1,4 +1,4 @@
 <h3 style="font-weight:300">You're popular!</h3>
 <p>
-  <a style="background:#3c7dc1;color:white;padding:5px 14px;border-radius:3px;text-decoration:none;display:inline-block;margin-top:4px;font-size:33px;" href='https://dev.to/notifications'>Read my notifications</a>
+  <a style="background:#3c7dc1;color:white;padding:5px 14px;border-radius:3px;text-decoration:none;display:inline-block;margin-top:4px;font-size:33px;" href='<%= app_url("/notifications") %>'>Read my notifications</a>
```

Comment:
`app_url(notifications_path)`

ps. you can check which routes we have using the following in the rails console:

```ruby
> show-routes -G notifications
                            notifications_counts GET      /notifications/counts(.:format)                                       notifications/counts#index
                             notifications_reads POST     /notifications/reads(.:format)                                        notifications/reads#create
                                   notifications GET      /notifications(.:format)                                              notifications#index
                                                 GET      /notifications/:filter(.:format)                                      notifications#index
                                                 GET      /notifications/:filter/:org_id(.:format)                              notifications#index
```